### PR TITLE
#19 - Package installation finalized

### LINKED ISSUE #19: HTTP support for installing package
Docs:

https://docs.microsoft.com/en-us/nuget/api/package-base-address-resource#download-package-content-nupkg
https://docs.microsoft.com/en-us/nuget/api/package-base-address-resource#download-package-manifest-nuspec

### PR DESCRIPTION
**Pull request by olegmoz**
Closes #19 
Enabled test for installation package via HTTP, project status updated

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#70](https://github.com/artipie/nuget-adapter/issues/70) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/3506839a-23dd-4534-9e5a-70c726258b1b); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artpie/nuget/RepositoryHttpIT.java [hunk 1]
```diff
@@ -142,4 +140,14 @@ private String runNuGet(final String... args) throws IOException, InterruptedExc
         Logger.debug(this, "Full stdout/stderr:\n%s", log);
         return log;
     }
+
+    private static String command() {
```
Comment (paulodamaso) [inline src/test/java/com/artpie/nuget/RepositoryHttpIT.java:144]: @olegmoz I don't think that we need to have this method, unless we are planning to use it more than once in the future, is that the case?

Comment (olegmoz) [reply src/test/java/com/artpie/nuget/RepositoryHttpIT.java:144]: @paulodamaso yes exactly, this is the case. I am planning to push package into the repository using CLI as well. I am sure other IT tests will follow and will use the same method

**Comment by paulodamaso** [comment]
@olegmoz That's good, thanks for the answer

**Comment by paulodamaso** [comment]
@g4s8 please merge

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@paulodamaso please make sure you will find at least 3 issues during next CR, thanks

**Comment by paulodamaso** [comment]
@sereshqua Ok, I will, thank you

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**